### PR TITLE
Diamond singularity armor

### PR DESCRIPTION
**Pull request by solidwaterslayer**

### CODE REVIEW

File: src/main/resources/data/minebound/recipes/gilded_diamond_helmet.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "###",
+    "# #",
+    "   "
```
Comment (MattKirchh) [inline src/main/resources/data/minebound/recipes/gilded_diamond_helmet.json:6]: Remove this third and empty empty string from all the helmet recipes, even the ones that were already done. Seems like they were missed in review.

File: src/main/resources/assets/minebound/lang/en_us.json [hunk 2]
```diff
@@ -56,6 +56,14 @@
   "item.minebound.illuminant_steel_chestplate": "Illuminant Steel Chestplate",
   "item.minebound.illuminant_steel_leggings": "Illuminant Steel Leggings",
   "item.minebound.illuminant_steel_boots": "Illuminant Steel Boots",
+  "item.minebound.gilded_diamond_helmet": "Gilded Diamond Helmet",
```
Comment (MattKirchh) [inline src/main/resources/assets/minebound/lang/en_us.json:59]: Missing Myrial Singularity entries.

File: src/main/java/com/mineboundteam/minebound/registry/ItemRegistry.java [hunk 3]
```diff
@@ -62,6 +62,10 @@ public class ItemRegistry {
...
     public static final RegistryObject<Item> MYRIAL_EFFIGY_LEGGINGS = ITEMS.register("myrial_effigy_leggings", () -> new MyrialArmorItem(ArmorMaterials.MYRIAL_EFFIGY, EquipmentSlot.LEGS, new Item.Properties().tab(MineBound.MINEBOUND_TAB), ArmorTier.EFFIGY, ArmorConfigRegistry.EFFIGY_LEGGINGS));
     public static final RegistryObject<Item> MYRIAL_EFFIGY_BOOTS = ITEMS.register("myrial_effigy_boots", () -> new MyrialArmorItem(ArmorMaterials.MYRIAL_EFFIGY, EquipmentSlot.FEET, new Item.Properties().tab(MineBound.MINEBOUND_TAB), ArmorTier.EFFIGY, ArmorConfigRegistry.EFFIGY_BOOTS));
+    public static final RegistryObject<Item> MYRIAL_SINGULARITY_HELMET = ITEMS.register("myrial_singularity_helmet", () -> new MyrialArmorItem(ArmorMaterials.MYRIAL_SINGULARITY, EquipmentSlot.HEAD, new Item.Properties().tab(MineBound.MINEBOUND_TAB), ArmorTier.SINGULARITY, ArmorConfigRegistry.SINGULARITY_HELMET));
```
Comment (MattKirchh) [inline src/main/java/com/mineboundteam/minebound/registry/ItemRegistry.java:65]: Singularity armor should be registered after the Synergy armor.

**Review by MattKirchh — CHANGES_REQUESTED** [review]

File: src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java [hunk 4]
```diff
@@ -15,9 +15,12 @@
...
     MYRIAL_SUIT(0, new int[] { 2, 5, 6, 2 }, 0, SoundEvents.ARMOR_EQUIP_IRON, 0f, 0f, null),
-    MYRIAL_SYNERGY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_DIAMOND, 0f, 0f, null), 
-    ENERGIZED_IRON(15, new int[]{2, 5, 6, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 0.5F, 0.0F,  Ingredient.of(ItemRegistry.ENERGIZED_IRON_INGOT.get())), 
-    ILLUMINANT_STEEL(30, new int[]{3, 5, 7, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 1.0F, 0.0F, Ingredient.of(ItemRegistry.ILLUMINANT_STEEL_INGOT.get()));
+    MYRIAL_SYNERGY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_DIAMOND, 0f, 0f, null),
+    MYRIAL_SINGULARITY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_NETHERITE, 0f, 0f, null),
+    ENERGIZED_IRON(15, new int[]{2, 5, 6, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 0.5F, 0.0F,  Ingredient.of(ItemRegistry.ENERGIZED_IRON_INGOT.get())),
+    ILLUMINANT_STEEL(30, new int[]{3, 5, 7, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 1.0F, 0.0F, Ingredient.of(ItemRegistry.ILLUMINANT_STEEL_INGOT.get())),
+    GILDED_DIAMOND(33, new int[] { 3, 6, 8, 3 }, 8, SoundEvents.ARMOR_EQUIP_DIAMOND, 2.5F, 0,  Ingredient.of(ItemRegistry.GILDED_DIAMOND_INGOT.get())),
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java:22]: make enchantability follow the tools and check those defense values. They shouldn't be copy and pasted. Both should also have some knockback resistance like .1 for gilded diamond and .3 for myrial. Myrial should also have higher toughness, like 3.5-4

File: src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java [hunk 5]
```diff
@@ -15,9 +15,12 @@
 public enum ArmorMaterials implements ArmorMaterial {
     MYRIAL_EFFIGY(0, new int[]{1, 4, 5, 2}, 0, SoundEvents.ARMOR_EQUIP_CHAIN, 0f, 0f, null),
     MYRIAL_SUIT(0, new int[] { 2, 5, 6, 2 }, 0, SoundEvents.ARMOR_EQUIP_IRON, 0f, 0f, null),
-    MYRIAL_SYNERGY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_DIAMOND, 0f, 0f, null), 
-    ENERGIZED_IRON(15, new int[]{2, 5, 6, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 0.5F, 0.0F,  Ingredient.of(ItemRegistry.ENERGIZED_IRON_INGOT.get())), 
-    ILLUMINANT_STEEL(30, new int[]{3, 5, 7, 2}, 8, SoundEvents.ARMOR_EQUIP_IRON, 1.0F, 0.0F, Ingredient.of(ItemRegistry.ILLUMINANT_STEEL_INGOT.get()));
+    MYRIAL_SYNERGY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_DIAMOND, 0f, 0f, null),
+    MYRIAL_SINGULARITY(0, new int[] { 3, 6, 8, 3 }, 0, SoundEvents.ARMOR_EQUIP_NETHERITE, 0f, 0f, null),
```
Comment (insertStringHere) [inline src/main/java/com/mineboundteam/minebound/item/ArmorMaterials.java:19]: This needs the armor toughness and knockback resistance of netherite

**Review by insertStringHere — CHANGES_REQUESTED** [review]

**Comment by MattKirchh** [comment]
Also, resolve merge conflicts.

**Review by MattKirchh — APPROVED** [review]

**Review by insertStringHere — APPROVED** [review]

**Pull request merged**